ContinuousML/NQ_DR_4_10_20_Ideal_27/2_Training_Workflow/params.yaml

```diff
@@ -7,4 +7,4 @@ train:
     activation: relu
     fc_layers: [512, 256, 128, 64]
 eval:
-    batch_size: 16
+    batch_size:   16
```

Comment:
We can enhance the hyperparameters here as teh model training process becomes more mature.